crates/integration-tests/Cargo.toml

```diff
@@ -68,6 +69,7 @@ features = ["full"]
 workspace = true
 
 [dev-dependencies]
+mimalloc = "0.1"
```

Comment:
why you need this in tests?

Reply:
ah wrong PR, adding it for the benchmarks to have more accurate results as we're using mimalloc in the gateway